src/kbmod/__init__.py

```diff
@@ -12,6 +12,8 @@
 from . import jointfit_functions
 from . import result_list
 from . import file_utils
+
+# import the filters subdirectory
 from . import filters
```

Comment:
I think you'll want to remove `src/kbmod/filters.py`

Reply:
It has been removed, but github is showing it as a file move into filters/stats_filter.py.